providers/oauth/oauth.httpobserver.js

```diff
@@ -67,7 +71,9 @@ FirefoxTabsAuth.prototype.launchAuthFlow = function (authUrl, stateObj, continua
 
   complete = function (location) {
     httpRequestObserver.unregister();
-    gBrowser.selectedTab = previousTab;
+    if (interactive) {
```

Comment:
Similar response to the freedom-for-chrome pull request:

All of these changes are being done to make Facebook reconnect a better UX. Facebook, unlike Google, doesn't let us get refresh (long-lived) tokens, so there is no way for us to get a valid Facebook token without opening an oauth tab. However, in most cases where we need to reconnect a user to Facebook, the user is going to be still logged into Facebook in their browser and still granting uProxy permission - this means that the auth tab will just open, redirect to uproxy.org with the token, then uProxy will close the tab. It is fairly annoying right now when this happens and the tab is focused on (a common scenario for me is I'm logged into FB on uProxy, lock my computer, come back and start doing unrelated stuff in Chrome, then suddenly a tab opens and closes).

What we want for uProxy is for this tab to open in the background (not be focused on). We've had this briefly working in Chrome where we override the core.oauth provider.

In the case of Firefox specifically, I've tested these changes manually in uProxy and verified that when we set interactive=true (for initial sign in), the tab is focused on, but when its false (for reconnect) the tab isn't focused